include/swift/AST/FineGrainedDependencies.h

```diff
@@ -19,14 +19,14 @@
 #include "swift/Basic/Range.h"
 #include "swift/Basic/ReferenceDependencyKeys.h"
 #include "llvm/ADT/Hashing.h"
+#include "llvm/ADT/SetVector.h"
 #include "llvm/Support/MD5.h"
 #include "llvm/Support/MemoryBuffer.h"
 #include "llvm/Support/YAMLParser.h"
 #include "llvm/Support/YAMLTraits.h"
 #include "llvm/Support/raw_ostream.h"
 #include <string>
 #include <unordered_map>
-#include <unordered_set>
```

Comment:
There's a usage of `std::unordered_set` on line 906

Reply:
Oops...